projects/packages/jetpack-mu-wpcom/src/common/tour-kit/components/keyboard-navigation.tsx

```diff
@@ -28,6 +28,7 @@ const KeyboardNavigation: React.FunctionComponent< Props > = ( {
 } ) => {
 	/**
 	 * Expand Tour Nav
+	 * @return {null} This component is non-rendering.
```

Comment:
I verified that in TS files it's also ok with something like this
```
	 * @return This component is non-rendering.
```
I have no opinion on whether you should make that sort of change here and in other TS files in this PR. 🙂

Reply:
No opinion myself either, but the extra doc shouldn't hurt here so I'll leave it.